docs/getting-started/configure.md

```diff
@@ -221,7 +221,7 @@ Edit each `config/<type>-<build>-<version>.pkrvars.hcl` files to configure the f
 machine hardware settings, as required:
 
 ```hcl linenums="1" title="config/linux-photon-5.pkrvars.hcl" hl_lines="1 6 9 13 27 32 36"
---8<-- "./builds/linux/ubuntu/24-04-lts/linux-ubuntu.pkrvars.hcl.example:10:100"
+--8<-- "./builds/linux/photon/5/linux-photon.pkrvars.hcl.example:10:100"
```

Comment:
Can you do a `make docs-serve-live` to ensure the correct lines are highlighted?

Reply:
```hcl linenums="1" title="config/linux-photon-5.pkrvars.hcl" hl_lines="1 6 9 13 27 32 36"
--8<-- "./builds/linux/photon/5/linux-photon.pkrvars.hcl.example:10:100" here is the lines